src/SourceBuild/tarball/patches/arcade/0009-Disable-Workloads-WiX-dependency-in-source-build.patch

```diff
@@ -0,0 +1,121 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Davis Goodin <dagood@microsoft.com>
+Date: Fri, 8 Oct 2021 16:15:42 -0700
+Subject: [PATCH] Disable Workloads WiX dependency in source-build
+
```

Comment:
Please add a patch comment linking to an issue tracking the Arcade issue - e.g. https://github.com/dotnet/arcade/issues/8014.